scripts/scr_fleet_advisor/scr_fleet_advisor.gml

```diff
@@ -112,6 +208,29 @@ function scr_fleet_advisor(){
                         cn.temp[119] = "";
                         if (obj_ini.ship_carrying[i] > 0) then cn.temp[119] = scr_ship_occupants(i);
                     }
+                    tooltip_draw($"Carrying ({cn.temp[118]}): {cn.temp[119]}");
+                    if (point_and_click([_goto_button.x1, _goto_button.y1, _goto_button.x2, _goto_button.y2])) {
+                        obj_controller.temp[40] = obj_ini.ship[i];
+                        with(obj_p_fleet) {
+                            for (var k = 0; k <= 40; k++) {
+                                if (capital[k] == obj_controller.temp[40]) then instance_create(x, y, obj_temp7);
+                                if (frigate[k] == obj_controller.temp[40]) then instance_create(x, y, obj_temp7);
+                                if (escort[k] == obj_controller.temp[40]) then instance_create(x, y, obj_temp7);
+                            }
+                        }
+                        if (instance_exists(obj_temp7)) {
```

Comment:
> if this is being redone can we please please deprecate the obj_temp7 usage

I would be glad, but I have no idea what `obj_temp7` is, or what it does, or how to deprecate this. I just copy-pasted the code chunk, so that it click detection is where the button coords are.

So if you know how to - you can do that and I cherry pick. Or create a separate PR, maybe?